package.json

```diff
@@ -41,6 +41,7 @@
   "dependencies": {
     "sequelize": "^3.30.2",
     "sequelize-cli": "^2.5.1",
+    "split.js": "1.3.5",
```

Comment:
Simple enough: Rather than removing the CSS classes, I just removed the background image settings and left the cursor as is.  The `grip` image is gone, visual cue remains, resize works without throwing any errors.